src/renderer/assets/less/5_partials/modal.less

```
@@ -18,12 +18,25 @@
 .modal {
   position: absolute;
   top: 0;
-  background: rgba(255,255,255,0.9);
+  background: rgba(0,0,0,0.25);
   width: 100vw;
   height: 100vh;
-  z-index: 2;
+  z-index: 5;
   &__dialog {
-    &--s {}
+    &--boxed {
+      border: 1px solid @mainLight;
+      border-radius: 4px;
+      background-color: @white;
+      .modal__actions {
+        display: block;
+        text-align: center;
+        padding-bottom: 0;
+      }
+    }
+    &--s {
```

It's hard to understand the meaning of `s`
```suggestion
    &--small {
```

---

`s` in not so self-explanatory for `height: auto`, could we have `&--small`?